config/initializers/floe_docker_runner.rb

```diff
@@ -0,0 +1,21 @@
+def floe_docker_runner
```

Comment:
@agrare I have a separate problem with this in that it is introducing a method at the top-level scope.  Can we move this method into a module/namespace (not sure if you can introduce `ManageIQ::Providers::Workflows` namespace here)?  Or perhaps move it as an engine method but keep it called here, such as `ManageIQ::Providers::Workflows::Engine.floe_docker_runner`?

Apologies in advance because I know my suggestions have been having you copy paste this all over creation :(

Reply:
I moved this method to the `Workflows::Engine` class